dune/grid/CpGrid.hpp

```diff
@@ -47,6 +47,12 @@
 // Warning suppression for Dune includes.
 #include <opm/common/utility/platform_dependent/disable_warnings.h>
 
+#include <dune/common/version.hh>
+
+#if HAVE_MPI && DUNE_VERSION_NEWER(DUNE_GRID, 2, 3)
```

Comment:
I suppose that ´DUNE_VERSION_NEWER(DUNE_GRID, 2, 3)` is superfluous because the build system hopefully already ensures this. If you like to keep it around, that's fine with me, too... (dune.module still needs to be modified to make this requirement explicit, though...)